go.mod

```diff
@@ -92,3 +91,92 @@ require (
 	k8s.io/client-go v0.20.1
 	k8s.io/kubectl v0.20.1
 )
+
+require (
```

Comment:
What is the difference between this `require` and the one above? Why do we need both?

Reply:
No difference, `go mod tidy generated 2 of them` I have merged them now